markdown_lab/core/throttle.py

```
@@ -26,4 +26,4 @@ def throttle(self) -> None:
         if time_since_last < self.min_interval:
             time.sleep(self.min_interval - time_since_last)
 
-        self.last_request_time = time.time()
+        self.last_request_time = time.time()
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)